crates/snark-acct-sys/src/handlers.rs

```diff
@@ -0,0 +1,25 @@
+use strata_acct_types::{AccountId, AcctResult, BitcoinAmount, MsgPayload};
+use strata_ledger_types::ISnarkAccountStateMut;
+use strata_snark_acct_types::MessageEntry;
+
+pub fn handle_snark_msg(
+    cur_epoch: u32,
+    snark_state: &mut impl ISnarkAccountStateMut,
+    from: AccountId,
+    payload: &MsgPayload,
+) -> AcctResult<()> {
+    let msg = MessageEntry::new(from, cur_epoch, payload.clone());
+    snark_state.insert_inbox_message(msg)?;
+    Ok(())
+}
+
+pub fn handle_snark_transfer(
+    _cur_epoch: u32,
+    _snark_state: &mut impl ISnarkAccountStateMut,
+    _from: AccountId,
+    _amt: BitcoinAmount,
+) -> AcctResult<()> {
+    // Nothing to do yet, the balance should be already updated.
+    // TODO: anything more to be done?
+    Ok(())
+}
```

Comment:
Why do we need this function?
If you want to keep this function, shouldn't we at least do some check if the balance was already update?